src/python/nimbusml/base_predictor.py

```diff
@@ -12,7 +12,6 @@
 
 from sklearn.base import BaseEstimator
 from sklearn.utils.multiclass import unique_labels
-from sklearn.utils.validation import check_is_fitted
```

Comment:
>sklearn.utils.validation import check_is_fitted [](start = 5, length = 47)

can you see if we pass test_estimator_checks now

Reply:
I think `test_estimator_checks` runs as part of the `--runTests` build argument and so if those tests had failed then it would have shown up during the CI run. Please correct me if I'm wrong.

I will run the tests locally to verify that they pass.

Reply:
`test_estimator_checks` passed when run locally.

Reply:
you right they passed as part of CI

---
In reply to: [305118075](https://github.com/microsoft/NimbusML/pull/195#discussion_r305118075) [](ancestors = 305118075)